lib/src/dotenv.dart

```diff
@@ -49,6 +49,19 @@ class DotEnv {
   /// Clear [env]
   void clean() => _envMap.clear();
 
+  String get(String name, {String? fallback}) {
+    final value = maybeGet(name, fallback: fallback);
+    assert(() {
```

Comment:
Does `assert(value != null || fallback != null,'A non-null fallback is required for missing entries');` do the same thing as line 54 to 59?

Reply:
It is the same (old habbits). I have made that change and added the readme/changelog updates.